### PR TITLE
Add More Classes and Scenarios mod.

### PR DESCRIPTION
**Pull request by Noctifer-de-Mortem**
I added this mod for two reasons.

1.) I had a few ideas I have wanted to add and have had trouble doing so in mainline line, a mod might be the way to go. (Bionic Spec-Ops and Fema-Death Camp).

2.) mugling expressed interest in moving some professions to a mod to do away with clutter in #18204.

> I'm starting to wonder if we have too many professions. Perhaps some should be moved to a mod?

This would be the mod to do so in as well.

### CODE REVIEW

File: data/mods/more_classes_scenarios/cs_start_locations.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "name" : "Fema Entrance",
+    "target" : "fema_entrance"
+  },
+  {
+    "type" : "start_location",
+    "ident" : "fema_s",
+    "name" : "Fema Camp",
+    "target" : "fema"
+  }
+]
```
Comment (mugling) [inline data/mods/more_classes_scenarios/cs_start_locations.json]: Minor issue: no newline at end of file

**Comment by mugling** [comment]
Less minor: dont use special characters eg. `&` in PR titles

File: data/mods/more_classes_scenarios/modinfo.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "type": "MOD_INFO",
+    "mod-type": "SUPPLEMENTAL",
+    "ident": "more_classes_scenarios",
+    "name": "More Classes and Scenarios.",
```
Comment (mugling) [inline data/mods/more_classes_scenarios/modinfo.json]: Avoid trailing dot

**Comment by mugling** [comment]
JSON load errors

**Comment by Noctifer-de-Mortem** [comment]
Corrected errors and mistakes, load error caused by incorrect item id.

**Comment by mugling** [comment]
@Noctifer-de-Mortem you can use `./cataclysm --check-mods` as an automated check for valid JSON

**Comment by mugling** [comment]
It looks like your heading towards being the maintainer for the professions. Some thoughts:

* Too many options for each tier with no real differentiation. For example poor student is strictly worse than janitor.

* The interface for selecting professions is horrible. We should work on this further.

* The more professions there are in `core` the greater the chance a future contribution will be rejected. If you want to add a profession to `core` try and move any inferior duplicates to mod in the same PR.

* Unique but useless items (clown shoes etc) don't add variety, only maintenance burden. Instead aim for meaningful differences or special abilities (skater is a good example with an inate bonus to doding on roller skates)

* If you want to add special abilities via code support ask a developer for help. It's likely to get support so long as it's balanced and reasonably practical to implement.

* Long term we should move to a system of learnable special abilities. The professions would be a good incubator for this.

**Pull request merged**